soco/core.py

```diff
@@ -1283,7 +1275,15 @@ def add_uri_to_queue(self, uri, position=0, as_next=False):
     @only_on_master
     def add_to_queue(self, queueable_item, position=0, as_next=False):
         """Adds a queueable item to the queue.
-        For Arguments see `add_uri_to_queue`"""
+
+        Args:
+            queueable_item (str): The item to be added to the queue
+            position (int): The index (1-based) at which the URI should be
+            added.
```

Comment:
I think the "added." was somehow left hanging. It should either be moved to the line above if that doesn't make it too long or indent it one more level to work correctly as Napoleon.